dewy/chunk/router.py

```diff
@@ -34,16 +34,17 @@ async def list_chunks(
     print(f"PerPage: {perPage}, Page: {page}, Limit: {limit}, Offset {offset}")
     results = await pg_pool.fetch(
         """
-        SELECT chunk.id, chunk.document_id, chunk.kind, TRUE as raw, chunk.text
-        FROM chunk
-        JOIN document ON document.id = chunk.document_id
-        WHERE document.collection_id = coalesce($1, document.collection_id)
-        AND chunk.document_id = coalesce($2, chunk.document_id)
-        ORDER BY chunk.id
+        SELECT k.id, k.document_id, k.kind, TRUE as raw, k.text
+        FROM chunk k
+        JOIN document d ON d.id = k.document_id
+        JOIN collection c ON d.collection_id = c.id
```

Comment:
The value coming in is collection name, not ID.

Reply:
True, but I'm not sure how this affects the code. It compares the value coming in (`$1`) to the collection name (`lower(c.name) = coalesce(lower($1), lower(c.name))`), and uses the resulting ID (`c.id`) to filter on `d.collection_id`.

Reply:
Added tests verifying this works as expected.